### PR TITLE
EIP-7732: Modify `ExecutionPayloadEnvelopeByRoot`

### PR DESCRIPTION
**Pull request by StefanBratanov**
- Made `ExecutionPayloadEnvelopeByRoot` plural to be consistent with other req/resp methods
- Renamed `MAX_REQUEST_PAYLOAD` to `MAX_REQUEST_PAYLOAD` to be consistent with `MAX_REQUEST_BLOCKS`
- Added `MAX_REQUEST_PAYLOADS` to mainnet/minimal configs

### CODE REVIEW

**Review by terencechain — APPROVED** [review]

**Review by potuz — APPROVED** [review]
LGTM

File: specs/_features/eip7732/p2p-interface.md [hunk 1]
```diff
@@ -37,6 +38,14 @@ This document contains the consensus-layer networking specification for EIP7732.
 |------------------------------------------|-----------------------------------|---------------------------------------------------------------------|
 | `KZG_COMMITMENT_INCLUSION_PROOF_DEPTH_EIP7732`   | `13` # TODO: Compute it when the spec stabilizes | Merkle proof depth for the `blob_kzg_commitments` list item |
 
+### Configuration
+
+*[New in EIP-7732]*
```
Comment (jtraglia) [inline specs/_features/eip7732/p2p-interface.md:43]: I would prefer it, but we generally don't use dashes in the "New in" comments.

```suggestion
*[New in EIP7732]*
```

File: specs/_features/eip7732/p2p-interface.md [hunk 2]
```diff
@@ -37,6 +38,14 @@ This document contains the consensus-layer networking specification for EIP7732.
...
+
+| Name                                     | Value                             | Description                                                         |
+|------------------------------------------|-----------------------------------|---------------------------------------------------------------------|
+| `MAX_REQUEST_PAYLOADS`   | `2**7` (= 128) | Maximum number of execution payload envelopes in a single request |
```
Comment (jtraglia) [inline specs/_features/eip7732/p2p-interface.md:47]: Table isn't formatted properly.
```suggestion
| Name                   | Value          | Description                                                       |
|------------------------|----------------|-------------------------------------------------------------------|
| `MAX_REQUEST_PAYLOADS` | `2**7` (= 128) | Maximum number of execution payload envelopes in a single request |
```

**Review by jtraglia — APPROVED** [review]
LGTM but with two small nits. Thanks!

**Comment by StefanBratanov** [comment]
> LGTM but with two small nits. Thanks!

Thanks, commited those.

**Pull request merged**